src/App.tsx

```diff
@@ -236,6 +322,54 @@ function App() {
             </InputRightElement>
           </InputGroup>
 
+          {/* TODO:
+          * * Test
+          * */}
+          <Heading mt={4} mb={1.5} size="sm">
+            Upload & Download 
+          </Heading>
+          <Text>
+            You can also upload with drag&drop and downlod with Ctrl + S
+          </Text>
+          <ButtonGroup 
+            size="sm"
+            display="flex"
+          >
+              <Button
+                size="sm"
```

Comment:
Minor, indentation seems slightly off here, could you run Prettier auto-formatting with `npm run format`? This should help your PR pass CI.